lib/handler/pool/worker.ex

```diff
@@ -0,0 +1,27 @@
+# This is a generic GenServer (ie GenGenServer 🤣)
+# This module is part of the implementation of the Handler.Pool functionality
+# and it should be considered an implementation detail which can change at any
+# time.
+# For now we are using poolboy and this module is specified as the worker_module
+# so we can pass arbitrary functions to be executed in the pool.
```

Comment:
I'm not sure I understand the purpose of this module. It's not used anywhere in this project, and it doesn't use any `Handler.Pool` functionality. Is the intention of using this as a stepping stone to convert over to `Handler.Pool` when we are ready?

Reply:
Nope you're right. In the earlier commits I was making use of `:poolboy` and I needed to have a worker module, but I got rid of that. I'll delete this module